categories.py

```diff
@@ -78,6 +78,29 @@
   "francescobrivio":  ["alca"],
   "malbouis":         ["alca"],
   CMSBUILD_USER:      ["tests" ],
+  # dpgs
+  "tsusa":            ["trk-dpg"],
+  "mmusich":          ["trk-dpg"],
+  "thomreis":         ["ecal-dpg"],
+  "mseidel42":        ["hcal-dpg"],
+  "georgia14":        ["hcal-dpg"],
+  "mileva":           ["muon-dpg"],
+  "battibass":        ["dt-dpg"],
+  "fcavallo":         ["dt-dpg"],
+  "namapane":         ["dt-dpg"],
+  "ptcox":            ["csc-dpg"],
+  "jhgoh":            ["rpc-dpg"],
+  "andresib":         ["rpc-dpg"],
+  "pavlov":           ["rpc-dpg"],
+  "kamon":            ["gem-dpg"],
+  "jlee":             ["gem-dpg"],
+  "fabferro":         ["ctpps-dpg"],
+  "jan-kaspar":       ["ctpps-dpg"],
+  "vavati":           ["ctpps-dpg"],
+  "rovere":           ["hgcal-dpg"],
+  "cseez":            ["hgcal-dpg"],
+  "fabiocos":         ["mtd-dpg"],
```

Comment:
@amassiro ,  this will remove @fabiocos from `operations` team/category. Either merge this line with https://github.com/cms-sw/cms-bot/blob/master/categories.py#L40 or remove line 40 if @fabiocos does not need `operations` rights.

Reply:
I have no strong preference, if @silviodonato and @qliphy prefer you may just merge the categories in the same line

Reply:
Thanks Shahzad. @amassiro I would prefer to keep `fabiocos` here, adding `operations`
```suggestion
  "fabiocos":         ["mtd-dpg","operations"],
```
 and deleting line 40 
 ```
   "fabiocos":         ["operations"],
  ```

Reply:
Of course, I did not realize this would overwrite the previous assignment. I will commit soon